reView/cli.py

```diff
@@ -129,3 +135,259 @@ def unpack_characterizations(
+
+    supply_curve_df = pd.read_csv(supply_curve_csv)
+    supply_curve_gdf = gpd.GeoDataFrame(
+        supply_curve_df,
+        geometry=gpd.points_from_xy(
+            x=supply_curve_df['longitude'], y=supply_curve_df['latitude']
+        ),
+        crs="EPSG:4326"
+    )
+
+    boundaries_gdf = gpd.read_file(boundaries)
+    boundaries_singlepart_gdf = boundaries_gdf.explode(index_parts=True)
+
+    boundaries_dissolved = boundaries_gdf.unary_union
+    background_gdf = gpd.GeoDataFrame(
+        {"geometry": [boundaries_dissolved]},
+        crs=boundaries_gdf.crs
+    ).explode(index_parts=False)
+
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore", category=UserWarning)
+        map_extent = background_gdf.buffer(0.01).total_bounds
+
+    map_vars = {
+        "total_lcoe": {
+            "breaks": [25, 30, 35, 40, 45, 50, 60, 70],
+            "cmap": 'YlGn',
+            "legend_title": "All-in LCOE ($/MWh)"
+        },
+        "mean_lcoe": {
+            "breaks": [25, 30, 35, 40, 45, 50, 60, 70],
+            "cmap": 'YlGn',
+            "legend_title": "Project LCOE ($/MWh)"
+        },
+        "lcot": {
+            "breaks": [5, 10, 15, 20, 25, 30, 35, 40, 50],
+            "cmap": 'YlGn',
+            "legend_title": "LCOT ($/MWh)",
+        }
+    }
+    if tech == "solar":
+        map_vars.update({
+            "capacity": {
+                "breaks": [100, 500, 1000, 2000, 3000, 4000],
+                "cmap": 'YlOrRd',
+                "legend_title": "Capacity (MW)"
+            }
+        })
+    elif tech == "wind":
+        supply_curve_gdf["capacity_density"] = (
+            supply_curve_gdf["capacity"] / supply_curve_gdf["area_sq_km"]
+        )
+        map_vars.update({
+            "capacity": {
+                "breaks": [100, 125, 150, 175, 200, 225],
+                "cmap": 'Blues',
+                "legend_title": "Capacity (MW)"
+            },
+            "capacity_density": {
+                "breaks": [2, 3, 4, 5, 6, 10],
+                "cmap": 'Blues',
+                "legend_title": "Capacity Density (MW/sq km)"
+            }
+        })
+    for map_var, map_settings in tqdm.tqdm(map_vars.items()):
+        g = plots.map_geodataframe_column(
+            supply_curve_gdf,
+            map_var,
+            color_map=map_settings.get("cmap"),
+            breaks=map_settings.get("breaks"),
+            map_title=None,
+            legend_title=map_settings.get("legend_title"),
+            background_df=background_gdf,
+            boundaries_df=boundaries_singlepart_gdf,
+            extent=map_extent,
+            layer_kwargs={"s": 1.25, "linewidth": 0, "marker": "o"},
+            legend_kwargs={
+                "marker": "s",
+                "frameon": False,
+                "bbox_to_anchor": (1, 0.5),
+                "loc": "center left"
+            }
+        )
+        plt.tight_layout()
+
+        out_png_name = f"{map_var}_{tech}.png"
```

Comment:
Would it be a lot of effort to allow the option of saving as a PDF (or any other extension that matplolib allows)? Vector graphics are often preferred over rasters in many applications, including journal publications